other/bootstrap_daemon/src/tox-bootstrapd.c

```diff
@@ -61,7 +61,7 @@ static int manage_keys(DHT *dht, char *keys_file_path)
 {
     enum { KEYS_SIZE = CRYPTO_PUBLIC_KEY_SIZE + CRYPTO_SECRET_KEY_SIZE };
     uint8_t keys[KEYS_SIZE];
-    FILE *keys_file;
+    FILE *keys_file = NULL;
```

Comment:
Shouldn't that be `nullptr`? We use `nullptr` elsewhere in this file.

Reply:
Done.